staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -201,17 +201,19 @@ type EFI struct {
 // +k8s:openapi-gen=true
 type ResourceRequirements struct {
 	// Requests is a description of the initial vmi resources.
-	// Valid resource keys are "memory" and "cpu".
+	// Valid resource keys are "memory", "memory-overhead" and "cpu".
 	// +optional
 	Requests v1.ResourceList `json:"requests,omitempty"`
 	// Limits describes the maximum amount of compute resources allowed.
-	// Valid resource keys are "memory" and "cpu".
+	// Valid resource keys are "memory", "memory-overhead" and "cpu".
 	// +optional
 	Limits v1.ResourceList `json:"limits,omitempty"`
-	// Don't ask the scheduler to take the guest-management overhead into account. Instead
-	// put the overhead only into the container's memory limit. This can lead to crashes if
-	// all memory is in use on a node. Defaults to false.
-	OvercommitGuestOverhead bool `json:"overcommitGuestOverhead,omitempty"`
```

Comment:
Hm, that breaks our API. Not sure how many use it. In general we said that we don't want to break the API at this stage. Could it be kept with forcing `memory-overhaed` to `zero`?

Reply:
Right, it does - that's why I tried to rush it in before committing to v1 but if we anyway want not to break the API then sure, we can deprecate it and apply the logic you suggest